__tests__/app.test.js

```diff
@@ -85,6 +85,95 @@ describe("/api/articles", () => {
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  describe("GET query sort_by", () => {
+    test("GET:200 sends an array of article objects sorted by date", () => {
+      return request(app)
+        .get("/api/articles?sort_by=created_at")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+    test("GET:200 sends an array of article objects sorted by author", () => {
+      return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSortedBy("author", { descending: true });
+        });
+    });
+    test("GET:400 sends an error message when given an invalid sort_by query", () => {
+      return request(app)
+        .get("/api/articles?sort_by=invalid")
+        .expect(400)
+        .then(({ body }) => {
+          const { message } = body;
+          expect(message).toBe("invalid query of sort_by");
+        });
+    });
+  });
+  describe("GET query order", () => {
+    test("GET:200 sends an array of article objects in asc order", () => {
+      return request(app)
+        .get("/api/articles?order=asc")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeSorted({ descending: false });
```

Comment:
I'd probably keep the same structure as above like: `toBeSortedBy("created_at", { ascending: true })` to make sure that the default query works